### PR TITLE
Added Read me

### PR DESCRIPTION
**Pull request by Abhishek022001**
ReadMe file has been added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+   git clone https://github.com/yassine-mdn/attch-reminder.git
+
+2. Navigate to the project directory:
+   cd attch-reminder
+
+3. Install the required dependencies:
+   pip install -r requirements.txt
+
+4. Run the application:
+   python main.py
```
Comment (yassine-mdn) [inline README.md]: its a chrome extension, not a python app

**Review by yassine-mdn — CHANGES_REQUESTED** [review]
the README.md doesn't match the app

**Comment by Abhishek022001** [comment]
Pls check now

**Review by yassine-mdn — APPROVED** [review]

**Comment by yassine-mdn** [comment]
close #1

**Pull request merged**